sound/soc/sof/intel/hda-dai.c

```diff
@@ -916,6 +916,17 @@ struct snd_soc_dai_driver skl_dai[] = {
 	},
 },
 #endif
+{
+	.name = "SOF Dummy",
```

Comment:
SOF echo reference?

Reply:
> SOF echo reference?

@plbossart I thought about this when I added the new cpu dai. The dai will be for echo reference only if I called it as "echo reference" and set capture capability only. But if we call it a dummy dai and set both playback and capture capability, we can use it for any cases that need a dummy cpu dai.
And we can use the dailink name to describe the purpose of the BE link. What do you think?